pkg/route/OWNERS

```diff
@@ -0,0 +1,19 @@
+reviewers:
+  - bparees
+  - smarterclayton
+  - mfojtik
+  - soltysh
+  - csrwng
+  - gabemontero
+  - adambkaplan
+  - ironcladlou
+  - Miciah
+  - danehans
+  - frobware
+approvers:
+  - bparees
+  - mfojtik
+  - soltysh
+  - adambkaplan
+  - ironcladlou
+  - Miciah
```

Comment:
do you really want me/adam/michal/maciej as approvers on this dir?  seems like it should be your team + group lead + architect.

Reply:
(most of us are here anyway, also:
https://github.com/openshift/openshift-controller-manager/blob/master/OWNERS#L9)

Reply:
I went ahead and changed it to match the OWNERS of the other repos we maintain